src/pages/library/Code.js

```diff
@@ -122,10 +134,15 @@ const Code = () => {
           div({ style: styles.header }, 'GATK4 Best Practices workflows'),
           div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
             _.map(method => {
-              const { namespace, name } = method
+              const { namespace, name, id } = method
+              const isMethodsRepoMethod = namespace && name
+              const href = isMethodsRepoMethod ?
+                `${getConfig().firecloudUrlRoot}/?return=${returnParam()}#methods/${namespace}/${name}/` :
+                `${getConfig().dockstoreUrlRoot}/workflows/${id.replace(/^#workflow\//, '')}`
```

Comment:
Kind of guessing on the Dockstore URL format. This seems to work.

Reply:
If Dockstore doesn't guarantee a contract regarding the URL format, I wonder if we should have a test that checks it so we can detect any breakages in the future  🤔

Reply:
While I didn't find see anything documented for URL formats, I'm not worried about it changing.

On further inspection, this looks consistent with how we handle Dockstore URLs elsewhere.
https://github.com/DataBiosphere/terra-ui/blob/f4329d39eaa8f7f8e6f8513a358a038851008268/src/libs/ajax.js#L176
https://github.com/DataBiosphere/terra-ui/blob/f4329d39eaa8f7f8e6f8513a358a038851008268/src/components/common.js#L368-L373